server.go

```diff
@@ -35,15 +36,16 @@ func (o ServerOptions) ClientsAddr() string {
 }
 
 type Server struct {
-	ServerOptions     ServerOptions
-	DbStore           store.Store
-	Services          map[string]Service
-	backendM          sync.RWMutex
```

Comment:
probably we missed this, it was meant for reloading from routerconfig.toml when sighup is sent to the server to update services from the toml file (maybe to be revisited later)

Reply:
yes this signal management is nowhere. Lets leave this up to be implemented in another PR.